src/uwtools/api/sfc_climo_gen.py

```diff
@@ -0,0 +1,42 @@
+"""
+API access to the uwtools sfc_climo_gen driver.
+"""
+from pathlib import Path
+from typing import Dict
+
+import iotaa
+
+from uwtools.drivers.sfc_climo_gen import SfcClimoGen
+
+
+def execute(
+    task: str,
+    config_file: Path,
+    batch: bool = False,
+    dry_run: bool = False,
+) -> bool:
+    """
+    Execute an sfc_climo_gen task.
+
+    If ``batch`` is specified, a runscript will be written and submitted to the batch system.
+    Otherwise, the forecast will be run directly on the current system.
+
+    :param task: The task to execute
+    :param config_file: Path to UW YAML config file
```

Comment:
I saw you removed a lot of the `uw yaml` language. In case you wanted to change this to just `YAML` here as well.

Reply:
Yes, thank you, it kind of goes without saying. Fixing.